config/src/config/secure_backend_config.rs

```diff
@@ -157,18 +157,17 @@ mod tests {
         };
 
         let text_from_config = "
-[vault]
-server = \"127.0.0.1:8200\"
-
-[vault.token]
-type = \"from_config\"
-token = \"test\"
+vault:
+    server: \"127.0.0.1:8200\"
+    token:
+        type: \"from_config\"
+        token: \"test\"
         ";
```

Comment:
Might be a little more friendly to use the raw string literal: https://doc.rust-lang.org/reference/tokens.html#raw-string-literals

Alternatively, I believe you might be able to `'` single quote them.

Reply:
Oh, will try that out!